controller/src/beerocks/master/db/db.cpp

```diff
@@ -2535,9 +2537,11 @@ bool db::set_client_stay_on_initial_radio(const sMacAddr &mac, bool stay_on_init
         node->client_initial_radio = network_utils::ZERO_MAC;
         // if enabling stay-on-initial-radio and client is already connected, update the initial_radio as well
     } else if (is_client_connected) {
-        auto bssid                 = node->parent_mac;
-        auto parent_radio_mac      = get_node_parent_radio(bssid);
-        node->client_initial_radio = tlvf::mac_from_string(parent_radio_mac);
+        auto bssid            = node->parent_mac;
+        auto parent_radio_mac = get_node_parent_radio(bssid);
+        auto initial_radio    = tlvf::mac_from_string(parent_radio_mac);
+        LOG(DEBUG) << "Setting client " << mac << " initial-radio to " << initial_radio;
+        node->client_initial_radio = initial_radio;
```

Comment:
```suggestion
        node->client_initial_radio = tlvf::mac_from_string(parent_radio_mac);
        LOG(DEBUG) << "Setting client " << mac << " initial-radio to " << node->client_initial_radio;
```
For what the extra string copy?
Same on line 4334.

Reply:
fixed both occurrences